Examples.md

```diff
@@ -0,0 +1,268 @@
+# Examples
+
+## Login
+
+```js
+import { SynapseClient } from 'synapse-react-client'
+
+SynapseClient.login('username', 'password').then(response => {
+  // session token available in response.sessionToken
+})
+```
+
+## Query a Synapse Table/View
+
+```js
+import { SynapseClient, SynapseConstants } from 'synapse-react-client'
+
+let request = {
+  entityId: 'syn123',
+  query: {
+    sql: 'SELECT * FROM syn123',
+    includeEntityEtag: true,
+    isConsistent: true,
+    offset: 0,
+    limit: 100,
+  },
+
+  partMask:
+    SynapseConstants.BUNDLE_MASK_QUERY_RESULTS |
+    SynapseConstants.BUNDLE_MASK_QUERY_COLUMN_MODELS |
+    SynapseConstants.BUNDLE_MASK_QUERY_SELECT_COLUMNS |
+    SynapseConstants.BUNDLE_MASK_QUERY_FACETS,
+}
+SynapseClient.getQueryTableResults(request, sessionToken)
+  .then(response => {
+    // query results are available
+  })
+  .catch(function (error) {
+    // handle Error (possibly an HTTPError)
+  })
+```
+
+## Markdown Rendering Example
+
+View the demo app incorporation of markdown [here](./src/demo/containers/App.tsx).
+
+To use the synapse markdown-it component you must pass it a wiki page id and an owner id. Additionally, you can configure an error view to display.
+
+| Props                                      | Explanation                                             |
+| ------------------------------------------ | ------------------------------------------------------- |
+| ownerId: String                            | ownerId for the synapse page                            |
```

Comment:
I wonder if this documentation should be moved to the component itself in a way where it is output in the styleguidist live demo, and then we just point people to that live demo instead of these examples (which may become stale without us knowing).